### PR TITLE
added indices meta-data and indices min() / Max()

### PR DESCRIPTION
**Pull request by hmb1**
Hi Charlie,
 added indices meta-data and min /max.
 nb in the the previous commit i put hyperspectral_indices_make.nco 
into root instead of the dir "hyperspectral" 
my Bad !!! if could remove this duplicate from repo thanks.
Youll need the latest version of NCO to run the script as it has a fix to "where" that can handle vpointers

The min/max tell an interesting story 
...Henry

### CODE REVIEW

**Comment by czender** [comment]
NCO 4.6.5-alpha02 has been tagged. Maybe @yanliu-chn can create the module, @max-zilla can verify extractor uses it, then we can do this PR.

**Comment by hmb1** [comment]
just pushed again - fixed a few typo's
...Henry

File: hyperspectral/hyperspectral_indices_make.nco [hunk 1]
```diff
@@ -1,294 +1,555 @@
...
+
+
+}
 
-iR900=min_coords(wavelength, 9e-7d);
-iR970=min_coords(wavelength, 9.7e-7d);
 
 
-// sanity check for wavelength coord  check min ,max, mode 
+/* sanity checks for required relectances -  make sure the are within resonable tolerance (1.0e-9) */
```
Comment (hmb1) [inline hyperspectral/hyperspectral_indices_make.nco]: HI David,
for i each index I can check if the required reflectances are available 
Would this be OK ?

File: hyperspectral/hyperspectral_indices_make.nco [hunk 2]
```diff
@@ -1,294 +1,555 @@
...
+   flg_dbg - if set then print debug info (default 0) 
+  
+   flg_rba - if set then calculate the averages after the ratio step (default 1)
+  
+   flg_mss - set the _FillValue 1.0e36 to all values in selected reflectances where value <=0.0f (default 1)
+
+   usage examples 
+   ncap2 -v -O  -S hyperspectral_indices_make.nco   in.nc out.nc                                         ( use default flags)     
+   ncap2 -v -O -s 'flg_dbg=1s;flg_rba=0s;flg_mss=1s;' -S hyperspectral_indices_make.nco   in.nc out.nc   ( set flags )
+
```
Comment (hmb1) [inline hyperspectral/hyperspectral_indices_make.nco]: Will add comments once we have nailed down the overall structure of script

File: hyperspectral/hyperspectral_indices_make.nco (hunk 1 above)
Comment (czender) [inline hyperspectral/hyperspectral_indices_make.nco]: @dlebauer I think @hmb1 is awaiting your reply to the above question

**Comment by czender** [comment]
@max-zilla this PR awaits your confirmation that workflow is using at least version NCO 4.6.5-alpha02

**Comment by max-zilla** [comment]
@czender just confirmed that is the module on Roger going forward. Will merge.

**Pull request merged**